### PR TITLE
Migrate OOI templates to @testing-library/react

### PR DESCRIPTION
**Pull request by amitdahan**
### 🔦 Summary
Other templates were migrated already, this marks OOI.

### 🗺️ Test Plan
Template tests keep passing.

### CODE REVIEW

File: packages/create-yoshi-app/templates/out-of-iframe/javascript/src/components/Settings/Settings.spec.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 import React from 'react';
-import { mount } from 'enzyme/build';
-import { Slider, ColorPickerColorSpace } from '@wix/wix-base-ui';
+import { render } from '@testing-library/react';
 import Settings from './Settings';
 
-describe('Settings', () => {
-  let wrapper;
+jest.mock('@wix/wix-base-ui', () => ({
+  ...jest.requireActual('@wix/wix-base-ui'),
```
Comment (deanshub) [inline packages/create-yoshi-app/templates/out-of-iframe/javascript/src/components/Settings/Settings.spec.js]: is this still necessary?

File: packages/create-yoshi-app/templates/out-of-iframe/typescript/src/components/Settings/Settings.spec.tsx [hunk 2]
```diff
@@ -1,11 +1,14 @@
 import * as React from 'react';
-import { mount } from 'enzyme/build';
-import { Slider, ColorPickerColorSpace } from '@wix/wix-base-ui';
+import { render } from '@testing-library/react';
 import Settings from './Settings';
 
-describe('Settings', () => {
-  let wrapper;
+jest.mock('@wix/wix-base-ui', () => ({
+  ...jest.requireActual('@wix/wix-base-ui'),
```
Comment (deanshub) [inline packages/create-yoshi-app/templates/out-of-iframe/typescript/src/components/Settings/Settings.spec.tsx]: same question

File: packages/create-yoshi-app/templates/out-of-iframe/javascript/src/components/Settings/Settings.spec.js (hunk 1 above)
Comment (amitdahan) [reply packages/create-yoshi-app/templates/out-of-iframe/javascript/src/components/Settings/Settings.spec.js]: Yup: 
https://github.com/wix/yoshi/blob/master/packages/create-yoshi-app/templates/out-of-iframe/typescript/src/components/Settings/Settings.tsx#L8-L9

File: packages/create-yoshi-app/templates/out-of-iframe/typescript/src/components/Settings/Settings.spec.tsx (hunk 2 above)
Comment (amitdahan) [reply packages/create-yoshi-app/templates/out-of-iframe/typescript/src/components/Settings/Settings.spec.tsx]: Yup:
https://github.com/wix/yoshi/blob/master/packages/create-yoshi-app/templates/out-of-iframe/typescript/src/components/Settings/Settings.tsx#L8-L9

**Pull request merged**